CCDFDataModel/CCDFHDF5IO.h

```diff
@@ -118,6 +123,171 @@ class CDFHDF5Reader : public CDFReader {
   bool b_KNMIHDF5UseEndTime;
 
 public:
+  class CustomVolScanReader : public CDF::Variable::CustomReader {
+  public:
+    ~CustomVolScanReader() {}
+
+    int getCalibrationParameters(CT::string formula, float &factor, float &offset) {
```

Comment:
This should go to the CPP file if no templates are used

Reply:
done